app/presenters/reports/repository_presenter.rb

```diff
@@ -44,7 +44,9 @@ def to_hash(region)
         bs_below_200_patients: bs_below_200_patients[slug],
         bs_below_200_rates: bs_below_200_rates[slug],
         bs_below_200_with_ltfu_rates: bs_below_200_rates(with_ltfu: true)[slug],
-        bs_below_200_breakdown: diabetes_treatment_outcome_breakdown(:bs_below_200)[slug],
+        bs_below_200_breakdown_rates: diabetes_treatment_outcome_breakdown_rates(:bs_below_200)[slug],
+        bs_200_to_300_breakdown_rates: diabetes_treatment_outcome_breakdown_rates(:bs_200_to_300)[slug],
```

Comment:
We added this earlier, but did some refactoring so that 
- bs_200_to_300_breakdown_rates
- bs_200_to_300_breakdown_rates 
are not required in the current PR. 

We are leaving them here, although. Since we'll  need them in a clean up task to add the breakdowns for BS >200 graph.